### DISCUSSION TITLE
Evento de Natal: Operação Polo Norte

### DISCUSSION THREAD

**Discussion by ghost** [leogianfagna/IssueTracker#910] [Ideia]
### Servidor afetado

Ausevento

### Qual o propósito da ideia?

Oferecer uma boa experiência e trazer uma nova dinâmica, onde o objetivo vai além de simplesmente derrotar inimigos, incentivando os jogadores a proteger algo.

### Descrição da ideia

Evento de Natal: Operação Polo Norte

A Oficina do Papai Noel está em perigo, e os jogadores precisam defendê-la de ondas de inimigos que querem destruir a Máquina de Presentes.

Objetivo: Os jogadores precisam proteger a Máquina de Presentes, que possui uma barra de vida. 
Se os inimigos chegarem perto, vão tentar atacá-la e causar dano. Se a máquina for destruída, todos os jogadores morrem, a fábrica explode, e uma mensagem de aviso aparece.

Equipamento: Antes de cada onda, os jogadores podem pegar armas (Espadas/picaretas/arcos/machados/marreta) e itens especiais pela Oficina do Papai Noel.

Itens especiais: itens "Lendários" (2 ou 3), que são mais fortes e têm encantamentos melhores que os itens normais do jogo. Esses itens ajudam os jogadores a sobreviver por mais tempo e incentivam uma competição para encontrá-los. 
(Ex: varinha magica melhorada da classe mago mob arena )

Desafios das Ondas:
Para não prolongar infinitamente e evitar ser uma atividade maçante, a ideia inicial são 50 ondas, com 5 chefes. 
A cada chefe completado, os jogadores recebem melhorias nos equipamentos, como acontece na mob arena a cada X ondas. Isso ajuda a enfrentar a dificuldade crescente.

Obs: Caso a ideia seja aprovada, pf não coloque creeper carregado '-'

### Aplicação

A ideia segue mecânicas da própria mob arena, com algumas modificações de mapa e jogabilidade (Defender algo).

### Referências

![image](https://github.com/user-attachments/assets/1ff0ace5-ac4d-4d51-aa5e-a5cd59b77bb7)

**Comment by leogianfagna** [comment]
@Yokiro27 gostaria de dizer que o micose desenvolveu o evento inteiro, com bosses, presentes e tudo mais. ficou incrivel. contudo, não tivemos tempo de realizar e espero que o proximo natal seja possivel.

estou organizando de uma forma melhor essas coisas para que possamos ter mais tempo de colocar em prática todas as ideias

**Reply by ghost** [reply]
Entendi, que pena! Mas tudo bem, imprevistos acontecem. Tenho certeza de que será uma ótima adição ao próximo evento de Natal.
